src/pps/pps.proto

```diff
@@ -670,6 +670,17 @@ message ListDatumRequest {
   bool reverse = 6;
 }
 
+// Emits a stream of datums as they are created from the given input. Client
+// must cancel the stream when it no longer wants to receive datums.
+message CreateDatumRequest {
+  // Input is the input to list datums from.
+  // The datums listed are the ones that would be run if a pipeline was created
+  // with the provided input.
+  Input input = 1;
```

Comment:
Is this set only on the first request? (so the request stream is like: `(input: { ... }, number: 100)...(number: 100)...(number: 100)`) or on every request? Could be good to add a comment clarifying this.

Reply:
That's correct, the input only matters on the first request. I'll make that clear in the comments.